src/coord/src/catalog/builtin.rs

```diff
@@ -709,7 +709,8 @@ lazy_static! {
             .with_named_column("position", ScalarType::Int64.nullable(false))
             .with_named_column("nullable", ScalarType::Bool.nullable(false))
             .with_named_column("type", ScalarType::String.nullable(false))
-            .with_named_column("default", ScalarType::String.nullable(true)),
+            .with_named_column("default", ScalarType::String.nullable(true))
+            .with_named_column("type_oid", ScalarType::Oid.nullable(false)),
```

Comment:
Does it make sense to add this column but not remove the existing `type` column? Should we just change `type` to be this instead of adding one?

Reply:
Whatever creates normalized data, I think--so unless these are names uniquely expressed here, using the OID's better IMO.

Reply:
The names are uniquely expressed here, because they use a different generating function than mz_types.